Compute time limit (#6191)
PipedreamHQ/pipedream: docs/docs/troubleshooting/README.md

If you're receiving this warning on a source or workflow, this means your allocated Credit Budget has been reached for the defined period.

You can increase this limit at any time in the [billing area of your settings](https://pipedream.com/settings/billing).

> ### Compute Time Limit Exceeded

Reviewer: ```suggestion
### Compute time limit exceeded
```

Reviewer: Changed "Time Limit Exceeded" to "time limit exceeded" to follow sentence case for headings (Google's Developer docs style guide)

> The Compute Time Limit only applies to **Free** accounts. 

Reviewer: ```suggestion
The compute time limit only applies to **free** accounts.
```

Reviewer: Changed "Compute Time Limit" to "compute time limit" and "Free" to "free" (Google's Developer docs style guide)

> The Compute TimeLimit restricts the amount of testing within the workflow builder, and the limit is refreshed every day.

Reviewer: ```suggestion
The compute time limit restricts the amount of testing within the workflow builder, and the limit is refreshed daily.
```

Reviewer: Changed "Compute TimeLimit" to "compute time limit" (Google's Developer docs style guide)

> For more information, please refer to the [Free Tier limits](/pricing#free-tier).

Reviewer: ```suggestion
For more information, please refer to the [Free tier limits](/pricing#free-tier).
```

Reviewer: Changed "Tier" to "tier" (Google's Developer docs style guide)
